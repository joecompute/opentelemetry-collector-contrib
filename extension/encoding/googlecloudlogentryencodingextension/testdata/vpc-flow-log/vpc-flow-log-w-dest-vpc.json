{
  "insertId": "29gom3ewhg88",
  "jsonPayload": {
    "reporter": "DEST",
    "src_location": {
      "city": "Indore",
      "country": "ind",
      "asn": 24560,
      "continent": "Asia",
      "region": "Madhya Pradesh"
    },
    "end_time": "2025-09-27T21:13:15.943125002Z",
    "start_time": "2025-09-27T21:13:15.686128981Z",
    "packets_sent": "90",
    "rtt_msec": "22",
    "dest_vpc": {
      "vpc_name": "default",
      "project_id": "elastic-obs-integrations-dev",
      "subnetwork_region": "asia-south1",
      "subnetwork_name": "default"
    },
    "dest_instance": {
      "zone": "asia-south1-c",
      "vm_name": "service-integration-dev-idc-ubuntu25-4",
      "region": "asia-south1",
      "project_id": "elastic-obs-integrations-dev"
    },
    "connection": {
      "src_ip": "122.168.194.41",
      "dest_ip": "10.160.0.36",
      "dest_port": 22,
      "protocol": 6,
      "src_port": 45004
    },
    "bytes_sent": "11590"
  },
  "resource": {
    "type": "vpc_flow_logs_config",
    "labels": {
      "name": "joe-test-flow-logs-default",
      "resource_container": "projects/774712120909",
      "location": "global"
    }
  },
  "timestamp": "2025-09-27T21:13:28.314589494Z",
  "labels": {
    "target_resource_name": "projects/774712120909/global/networks/default"
  },
  "logName": "projects/elastic-obs-integrations-dev/logs/networkmanagement.googleapis.com%2Fvpc_flows",
  "receiveTimestamp": "2025-09-27T21:13:28.314589494Z"
}