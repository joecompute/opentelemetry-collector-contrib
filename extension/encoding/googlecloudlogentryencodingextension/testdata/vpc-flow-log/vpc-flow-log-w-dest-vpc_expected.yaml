resourceLogs:
  - resource:
      attributes:
        - key: gcp.project
          value:
            stringValue: elastic-obs-integrations-dev
        - key: cloud.resource_id
          value:
            stringValue: networkmanagement.googleapis.com%2Fvpc_flows
        - key: gcp.resource_type
          value:
            stringValue: vpc_flow_logs_config
        - key: gcp.label.name
          value:
            stringValue: "joe-test-flow-logs-default"
        - key: gcp.label.resource_container
          value:
            stringValue: projects/774712120909
        - key: gcp.label.location
          value:
            stringValue: global
    scopeLogs:
      - logRecords:
          - attributes:
              - key: log.record.uid
                value:
                  stringValue: 29gom3ewhg88
              - key: gcp.label.target_resource_name
                value:
                  stringValue: projects/774712120909/global/networks/default
              - key: network.protocol.name
                value:
                  stringValue: tcp
              - key: source.address
                value:
                  stringValue: 122.168.194.41
              - key: destination.address
                value:
                  stringValue: 10.160.0.36
              - key: source.port
                value:
                  intValue: "45004"
              - key: destination.port
                value:
                  intValue: "22"
              - key: gcp.vpc.flow.reporter
                value:
                  stringValue: DEST
              - key: gcp.vpc.flow.network.rtt_ms
                value:
                  intValue: "22"
              - key: gcp.vpc.flow.bytes_sent
                value:
                  intValue: "11590"
              - key: gcp.vpc.flow.packets_sent
                value:
                  intValue: "90"
              - key: gcp.vpc.flow.start_time
                value:
                  intValue: "1759007595686128981"
              - key: gcp.vpc.flow.end_time
                value:
                  intValue: "1759007595943125002"
              - key: gcp.vpc.flow.destination.instance.project.id
                value:
                  stringValue: elastic-obs-integrations-dev
              - key: gcp.vpc.flow.destination.instance.vm.region
                value:
                  stringValue: asia-south1
              - key: gcp.vpc.flow.destination.instance.vm.name
                value:
                  stringValue: service-integration-dev-idc-ubuntu25-4
              - key: gcp.vpc.flow.destination.instance.vm.zone
                value:
                  stringValue: asia-south1-c
              - key: gcp.vpc.flow.destination.project.id
                value:
                  stringValue: elastic-obs-integrations-dev
              - key: gcp.vpc.flow.destination.subnet.name
                value:
                  stringValue: default
              - key: gcp.vpc.flow.destination.subnet.region
                value:
                  stringValue: asia-south1
              - key: gcp.vpc.flow.destination.vpc.name
                value:
                  stringValue: default
              - key: gcp.vpc.flow.source.asn
                value:
                  intValue: "24560"
              - key: gcp.vpc.flow.source.geo.city
                value:
                  stringValue: Indore
              - key: gcp.vpc.flow.source.geo.continent
                value:
                  stringValue: Asia
              - key: gcp.vpc.flow.source.geo.country.iso_code.alpha3
                value:
                  stringValue: ind
              - key: gcp.vpc.flow.source.geo.region
                value:
                  stringValue: Madhya Pradesh
            body: {}
            observedTimeUnixNano: "1759007608314589494"
            timeUnixNano: "1759007608314589494"
        scope: {}
